src/custom/utils/format.ts

```diff
@@ -0,0 +1,7 @@
+import BigNumber from 'bignumber.js'
+
+const TEN = new BigNumber(10)
+
+export function formatAtoms(amount: string, decimals: number): string {
+  return new BigNumber(amount).div(TEN.pow(decimals)).toString(10)
+}
```

Comment:
Sure there's no helper function to do that already?
I could swear I saw this somewhere...